Sources/SchemaStructure.swift

```diff
@@ -0,0 +1,5 @@
+public struct SchemaStructure {
```

Comment:
If you do think it's necessary, I would name it StructureSchema and make it mappable like ObjectSchema and ArraySchema